src/gc-pages.c

```diff
@@ -24,6 +24,9 @@ static size_t current_pg_count = 0;
 
 void jl_gc_init_page(void)
 {
+    if (jl_options.region_pg_cnt >= MIN_REGION_PG_COUNT) {
```

Comment:
It seems memory exception will be thrown if `pg_cnt <= MIN_REGION_PG_COUNT` in the function `jl_gc_alloc_region`. (Need to change `>=` to `>` in the above line)